governance/executive-council.md

```diff
@@ -0,0 +1,262 @@
+# pyOpenSci Executive Council
+
+The pyOpenSci Executive Council (EC) has four primary functions:
+
+1. **Guides the Organization**: The Executive Council defines and directs pyOpenSci’s high-level [mission, vision, and values](mission-values) and sets its strategic direction.
+
+2. **Advocates for pyOpenSci**: The Executive Council serves as an advocate and ambassador for the organization, leveraging members’ networks to enhance its reputation and support fundraising efforts.
+
+3. **Provides Strategic Insight**: The Executive Council offers perspective and guidance on major strategic decisions to ensure the organization’s long-term success.
+
+4. **Holds the [Executive Director](executive-director) accountable**: The Executive Council holds the Executive Director accountable while also providing high-level strategic and financial guidance and oversight.
+
+The Executive Director is currently a voting member of the Executive Council for items unrelated to their salary and accountability.
+
+The Executive Council holds broad expertise in building and running a scientific / data science nonprofit organization. It is complemented by the [Advisory Council](advisory-council), which offers domain-specific expertise, guidance, and input.
```

Comment:
do we even need 'scientific / data science' here? I could imagine very good EC members from other domains

Reply:
I agree, I don't think we should include that.